mods/braindance_protocol/lang/lang.lua

```diff
@@ -3,4 +3,5 @@ return {
   { id = "cn", name = "中文 (Chinese)"},
   { id = "da", name = "Dansk (Danish)"},
   { id = "ru", name = "Русский (Russian)"}
```

Comment:
Need a comma at the end of this line

Reply:
Sorry, missed that. Should be fixed, now.